### DISCUSSION TITLE
It stopped and won't prompt download

### DISCUSSION THREAD

**Discussion by piot-admin** [davetapley/helium-tax#33] [General]
not sure why, but it just stops calculating transactions and the prompt to download doesn't show so I'm not sure if it's still calculating or it's a bug.. The window is open for more than 30mins now. I've also tried other browsers and even incognito..

**Comment by davetapley** [comment]
You're not the first to see this 😞 
Can you follow steps and post results here ⬇️, thanks 🙏🏻 
https://github.com/davetapley/helium-tax/discussions/31#discussioncomment-1387722
